website/docs/modules/make_cohort_vcf.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Make Cohort VCF
```

Comment:
I know we haven't fixed this on the README yet so it's fine to leave as-is for now but just a heads up that we split MakeCohortVcf into 5 sub-workflows so this will need to change to reflect that soon. How complicated will it be to add new modules or change module names with this website organization?

Reply:
We can add/remove files as needed at this step.